File: ManageCertificate/Entites/RefOfficeInventory.cs
using System;
using System.Collections.Generic;

namespace Entites;

public partial class RefOfficeInventory
{
    public int Id { get; set; }

    public int? Inventory { get; set; }

    public int? Year { get; set; }

    public int? CertificateId { get; set; }

    public virtual RefCertificateType? Certificate { get; set; }
}

File: ManageCertificate/ManageCertificate/wwwroot/client/src/Models/RefOfficeInventory.ts
export interface RefOfficeInventory {
  id: number;
  inventory: number;
  year: number;
  certificateId: number;
  certificateName?: string;          // להוסיף שדה אם קיים
  unusedInventoryBalance?: number;   // להוסיף שדה אם קיים
  editedMinimum?: number;
  minimum?: number;
  }
  